nl_NL/LC_MESSAGES/login.po

```diff
@@ -253,11 +253,11 @@ msgid "EMAIL_VERIFICATION_ADDRESS_REMOVED"
 msgstr "Sorry daarvoor! We hebben gemarkeerd dat je e-mailadres niet klopt, dus je hoort vanaf nu geen e-mails meer van Sudomemo te ontvangen."
 
 msgid "THANKS_FOR_LOGIN_PROMPT_INTRO"
-msgstr "Welcome to Sudomemo! Now, let's write a little message to say who you are! (This will be posted as a Flipnote in our Introductions channel.)"
+msgstr "Welkom op Sudomemo! Nu, Laten we een klein berichtje achter om te zeggen wie je bent! (Dit zal worden gepost als een flipnote in de "introduction" kanaal.)"
```

Comment:
The first letters of "Flipnote" and "Introductions" should be capitalized. Also Introduction__s__ should be plural.

Reply:
If quotation marks must be used here, they have to have \ before them. It would be best if they weren't used at all, though.